com.github.vikdevelop.googlemaps.yml

```
@@ -0,0 +1,58 @@
+app-id: com.github.vikdevelop.googlemaps
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions: 
+  - org.freedesktop.Sdk.Extension.node16
+command: run-a.sh
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+
+build-options:
+  append-path: /usr/lib/sdk/node14/bin
```

```suggestion
  append-path: /usr/lib/sdk/node16/bin
```

Since you use the node16 SDK extension.